### ISSUE TITLE
Include "vital" upstream status information in replies

### ISSUE THREAD

**Issue by j6k4m8** [bookwyrm-social/bookwyrm#2865]
**Is your feature request related to a problem? Please describe.**

Here's a (doctored) screenshot I took on May 30 2023 on the bookwyrm.social instance:

<img width="577" alt="image" src="https://github.com/bookwyrm-social/bookwyrm/assets/693511/73729e19-50f4-468c-8b0a-bdde01ea6924">

This was in response to `other user`'s status regarding a progress update, but I can't tell what book it was about just from this comment.

**Describe the solution you'd like**

 Would love to amend the title to something like 

> `so and so` replied to `other user`'s status on "This is how you lose the time war"

**Describe alternatives you've considered**

...good alternative is just clicking through :)

**Additional context**

Forgive the very red screenshot; i don't want to screenshot others' posts without permission but can't replicate this particular behavior as a single user:

![image](https://github.com/bookwyrm-social/bookwyrm/assets/693511/d4ca10fd-5773-409d-a717-ae925eda7212)

The status at the very bottom is the relevant post that I'm showing up top; the relevant upshot is that the upstream status is a reply itself, and so it seems like we'd have to walk back up recursively to actually find out the title of the book. another option is to just show a little version of the previous comment inline in the timeline, on the same "grapevine" that this chained screenshot shows.

**Comment by mouse-reeve** [comment]
I agree! There's some similar logic in notifications: https://github.com/bookwyrm-social/bookwyrm/blob/main/bookwyrm/templates/notifications/items/reply.html#L23
